reference/7.3/Microsoft.PowerShell.Core/About/about_Language_Modes.md

```diff
@@ -14,66 +14,170 @@ Explains language modes and their effect on PowerShell sessions.
+
+```powershell
+(Get-PSSessionConfiguration -Name Test).LanguageMode
+FullLanguage
+```
+
+On other session configurations, you can find the language mode indirectly by
+finding the language mode of a session that's created using the session
 configuration.
 
-All PowerShell sessions have a language mode, including PSSessions that you
-create by using the `New-PSSession` cmdlet, temporary sessions that use the
-ComputerName parameter, and the default sessions that appear when you start
-PowerShell.
+> [!NOTE]
+> Session configurations are a feature of WSMan-based PowerShell remoting.
+> They're used only when you use the `New-PSSession`, `Invoke-Command`, or
+> `Enter-PSSession` cmdlets to connect to a remote Windows computer. The
+> `Get-PSSessionConfiguration` cmdlet is only available on Windows computers.
 
-Remote sessions are created by using the session configurations on the remote
-computer. The language mode set in the session configuration determines the
-language mode of the session. To specify the session configuration of a
-PSSession, use the ConfigurationName parameter of cmdlets that create a
-session.
+## Setting the language mode
+
+The language mode in a PowerShell session can be set through the built in
+`$ExecutionContext` variable.
+
+```powershell
+$ExecutionContext.SessionState.LanguageMode = "ConstrainedLanguage"
+```
+
+However, doing this is only useful for experimenting with language modes.
+Language modes are intended to provide added security to PowerShell sessions
+for specific contexts.
+
+Languages modes are set when you use a system application control policy or
+create a session configuration.
+
+### Using a system application control policy
+
+PowerShell automatically runs in `ConstrainedLanguage` mode when it's running
+under a system application control policy. The only application control
+policies detected are AppLocker and Windows Defender Application Control (WDAC)
+on Windows platforms.
+
+PowerShell applies other restrictions besides language modes when it detects an
+application control policy. For example, there are additional restrictions to
+dot-sourcing and module importing under a policy.
+
+When a PowerShell session is started under a policy, it runs in
+`ConstrainedLanguage` mode. Beginning in PowerShell 7.4, the start up banner
+displays the that it's running in that mode. This allows users to have a usable
+interactive shell experience, running cmdlets and native commands, as well as
+access to basic language elements. But the user can't access PowerShell, .NET,
+or COM APIs that could be abused by a malicious actor.
+
+Any script or script-based module executed in this session run in
+`ConstrainedLanguage` mode. However, any script or script-based module allowed
+by the policy runs in `FullLanguage` mode without any constraints. This way, a
+system locked down by policy can have scripts that are trusted by the policy
+and run without constraints.
```

Comment:
```suggestion
and run with few constraints.
```

Reply:
The dot-sourcing across language modes seems the most common constraint encountered.

Reply:
Also restrictions in module importing.  Not technically language element, but users probably don't care about that.